src/include/mpir_request.h

```diff
@@ -357,9 +357,6 @@ static inline MPIR_Request *MPIR_Request_create(MPIR_Request_kind_t kind, int po
 #define MPIR_Request_add_ref(req_p_) \
     do { MPIR_Object_add_ref(req_p_); } while (0)
 
-#define MPIR_Request_release_ref(req_p_, inuse_) \
```

Comment:
We have a lot of these wrappers, though not consistently applied to all object types. For instance group, attr, comm, errhandler, and datatype all have some version of add/release_ref macros. I think a cleanup is worthwhile, but it should be more complete. Perhaps to defer this in order to stay on fast track.

Reply:
OK